### ISSUE TITLE
local feature detector

### ISSUE THREAD

**Issue by JustWon** [JuanDuGit/DH3D#2]
According to the paper, there are two the matchability score maps (S and S') in local feature detector.

![image](https://user-images.githubusercontent.com/14357526/91639228-1511e900-ea50-11ea-9f76-49225af772d9.png)

But I found out that there is no use of score map S' in the detector loss function.
![image](https://user-images.githubusercontent.com/14357526/91639257-3e327980-ea50-11ea-8349-b0f19f756fe3.png)

And in the source code, you calculated the score but didn't use.
![image](https://user-images.githubusercontent.com/14357526/91639335-c4e75680-ea50-11ea-8ab9-a9e08af8239e.png)

Is the score S' unnecessary?

**Comment by JuanDuGit** [comment]
Hi, you're correct. S' is not used in the detector loss function. Since we use a siamese network to train the local descriptor and detector,  our model will produce the score maps for both P (S)  and  P'  (S') no matter if they are used during training.
